docs/ModuleWidgets.md

```diff
@@ -0,0 +1,39 @@
+## Module Widgets
+
+Since LORIS 23.0, LORIS incorporates a concept of "widgets" which
+modules can use to dynamically display graphical widgets in other
+modules when installed and activated. This is a powerful way to
+extend the capabilities of other modules within a new module.
```

Comment:
```suggestion
modules when installed and activated.
```

Not a big deal. I think it can be removed though. Reads more like a sales pitch than as providing useful info to the developer reading it